# 错误信息

```
Traceback (most recent call last):
  File "D:\Code\study_uiautomator2\demo-01-start.py", line 1, in <module>
    import uiautomator2 as u2
ModuleNotFoundError: No module named 'uiautomator2'
```

# 解决方法

* pyCharm存在一个自己独立的解释器，pyCharm包独立于系统python包
* 将解释器环境改为系统python位置即可

# 参考

* [Pycharm中如何设置Python的执行环境](http://www.coozhi.com/youxishuma/g4/69871.html)